pkg/v1/tkg/managementcomponents/management_component_install.go

```diff
@@ -48,18 +61,207 @@ type ManagementComponentsInstallOptions struct {
 	ManagementPackageRepositoryOptions ManagementPackageRepositoryOptions
 }
 
+func generateAddonSecretName(clusterName, addonName string) string {
+	return fmt.Sprintf("%s-%s-addon", clusterName, fmt.Sprintf("tanzu-%s", addonName))
+}
+
+func pauseAddonSecretReconciliation(ctx context.Context, clusterClient clusterclient.Client, addonSecreteName, namespace string) error {
+	log.Infof("Pausing reconciliation for %s/%s secret", namespace, addonSecreteName)
+	secret := &corev1.Secret{}
+	jsonPatch := []map[string]interface{}{
+		{
+			"op":    "add",
+			"path":  fmt.Sprintf("/metadata/annotations/tkg.tanzu.vmware.com~1addon-paused"),
```

Comment:
What does `~1` mean here?

Reply:
The "~1" is as special character used to "scape" the "/" int he real path.
See https://www.rfc-editor.org/rfc/rfc6901#section-3